src/dsp_tools/commands/xmlupload/stash/construct_and_analyze_graph.py

```diff
@@ -52,6 +52,9 @@ def _create_info_from_xml_for_graph_from_one_resource(
                 resptr_links.extend(_create_resptr_link_objects(resource.attrib["id"], prop))
             case "text-prop":
                 xml_links.extend(_create_text_link_objects(resource.attrib["id"], prop))
+            case "hasComment" | "hasDescription":
+                if xml_link := _create_text_link_object_from_special_tags(resource.attrib["id"], prop):
+                    xml_links.append(xml_link)
```

Comment:
@BalduinLandolt @Nora-Olivia-Ammann I have added a new commit that you perhaps want to review: https://github.com/dasch-swiss/dsp-tools/pull/1101/commits/d8630a0d63196d56f9afed2e082bd2c88f027d84

Reply:
lgtm